docs/endaq/ide_usage.rst

```diff
@@ -0,0 +1,685 @@
+############################
+``endaq.ide`` Usage Examples
+############################
```

Comment:
...so in *theory* I like this change. However the docs render the literal code in pink, so this turns into a Dunkin Donuts-colored logo:

![image](https://user-images.githubusercontent.com/5862803/146593310-7e55cca9-218a-45e9-beca-529f5d250064.png)

which personally, I pretty strongly dislike.

In the future we might want David Manion to look into changing the color scheme to avoid this, but for now if it were up to me I'd prefer this just being taken out (see suggestion below)
```suggestion
##########################
`endaq.ide` Usage Examples
##########################
```

Reply:
I would really rather not remove it. The name should appear as code; the coloring of the name in headings is controlled by the CSS. Single-backtick is emphasis, not code font. So, until the CSS gets updated, either the color won't match or the font will be wrong. I think the former is better.

Reply:
A minor addition to the stylesheet renders it like this:
![image](https://user-images.githubusercontent.com/14080057/146604715-baa82361-5511-42f9-a8f1-b14bb18e924f.png)

This is the addition:
```css
h1 > code > span.pre {
  color: var(--pst-color-h1);
}
```
Unfortunately, the CSS isn't in the repo.

Reply:
yeah I'm looking forward to getting that CSS updated. Since that involves no code changes, I'll say it's fine